decred/decred/dcr/rpc.py

```diff
@@ -943,6 +1171,120 @@ def get(k, obj):
     return obj[k] if k in obj else None
 
 
+class COut:
+    def __init__(
+        self, addr, commitAmt, changeAddr, changeAmt,
+    ):
+        """
+        addr (str) Address to send sstx commit.
+        commitAmt (int) Amount to commit.
+        changeAddr (str) Address for change.
+        changeAmt (int) Amount for change.
+        """
+        self.addr = addr
+        self.commitAmt = commitAmt
+        self.changeAddr = changeAddr
+        self.changeAmt = changeAmt
+
+    def toJSON(self):
+        """
+        Encode the COuts to JSON.
+
+        Returns:
+            JSON: The encoded couts.
+        """
+        return {
+            "addr": self.addr,
+            "commitamt": self.commitAmt,
+            "changeaddr": self.changeAddr,
+            "changeamt": self.changeAmt,
+        }
+
+
+class DecodeScriptResult:
+    """DecodeScriptResult"""
```

Comment:
If not just for consistency with other **rpc** return type classes, use `decodescript` here to reference the actual RPC call. Alternatively, feel free to add an actual description of the type.